docs/install/java_setup.md

```diff
@@ -0,0 +1,111 @@
+# Setup the MXNet Package for Java
```

Comment:
Please add a preview link to this in your description:
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-13289/3/install/java_setup.html

Reply:
Will add new link once build finishes